warplib/src/warp_memory/accessors.cairo

```diff
@@ -0,0 +1,118 @@
+use array::Array;
+use array::ArrayTrait;
+use integer::u32_to_felt252;
+use integer::u256_from_felt252;
+use option::OptionTrait;
+use serde::Serde;
+use traits::Into;
+
+use warplib::warp_memory::WarpMemory;
+use warplib::warp_memory::WarpMemoryTrait;
+use warplib::warp_memory::WarpMemoryImpl;
+
+
+trait WarpMemoryAccesssorTrait {
+    fn store<T, impl TSerde: Serde<T>, impl TDrop: Drop<T>>(ref self: WarpMemory, position: felt252, value: T);
+    fn retrieve<T, impl TSerde: Serde<T>, impl TDrop: Drop<T>>(ref self: WarpMemory, position: felt252, size: felt252) -> T;
+    fn create<T, impl TSerde: Serde<T>, impl TDrop: Drop<T>>(ref self: WarpMemory, value: T);
+}
+
+trait WarpMemoryMultiCellAccessorTrait {
+    fn write_multiple(ref self: WarpMemory, position: felt252, ref value: Array::<felt252>);
+    fn read_multiple(ref self: WarpMemory, position: felt252, size: felt252) -> Array::<felt252>;
+}
+
+
+impl WarpMemoryMultiCellAccessor of WarpMemoryMultiCellAccessorTrait {
```

Comment:
can we write this `WarpMemoryMultiCellAccessor`  -> `WarpMemoryMultiCellAccessorImpl` as rather , not matter gonna anyway but let's keep this as a naming convention for traits, wdyt ?